spec/collections_publisher/archiving_child_topic_spec.rb

```diff
@@ -0,0 +1,70 @@
+feature "Archiving a child topic on Collections Publisher", collections_publisher: true do
+  include CollectionsPublisherHelpers
+
+  let(:topic_title) { title_with_timestamp }
+  let(:topic_slug) { slug_with_timestamp }
+
+  let(:child_slug) { slug_with_timestamp }
+  let(:child_title) { title_with_timestamp }
+  let(:link) { ["/topic", topic_slug, child_slug].join('/') }
+
+  scenario "Archiving a child topic" do
+    given_there_is_a_published_child_topic
+    when_i_archive_it
+    then_i_am_redirected_on_gov_uk
+  end
+
+  private
+
+  def given_there_is_a_published_child_topic
+    @published_parent_url = create_and_publish_parent_topic
+    create_and_publish_child_topic
+  end
+
+  def when_i_archive_it
+    click_link("Archive topic")
+    select2(topic_title, from: "Choose a topic to redirect to")
+    click_button "Archive and redirect to a topic"
+  end
+
+  def then_i_am_redirected_on_gov_uk
+    click_link(child_title)
```

Comment:
I'm a bit confused about the link we're clicking here - isn't the child archived at this point?